app/assets/javascripts/Components/groups_manager.jsx

```diff
@@ -570,6 +639,14 @@ const StudentsTable = withSelection(RawStudentsTable);
 
 class GroupsActionBox extends React.Component {
   render = () => {
+    var showHiddenTooltip = "";
+    if (this.props.hiddenStudentsCount && this.props.hiddenGroupsCount) {
```

Comment:
I'm not sure if you mean to exclude 0 here or just `""` values, but I think we *should* show the tooltips when one or both of these counts are 0.

But also `null` is a better default value for these props than `""` (what you're setting above)